src/BlockReportSearch/Block/BlockUsageTable.php

```diff
@@ -487,6 +493,33 @@ public function column_block_attrs($item): string
         );
     }
 
+    /**
+     * Add the number of columns to the table.
+     *
+     * This is a temporary fix to identify problems with the columns migrated here:
+     * https://jira.greenpeace.org/browse/PLANET-7489
+     * We need this function to identify P4 columns blocks using more than 4 columns.
+     *
+     * @param array $item Item.
+     * @phpcs:disable SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint
+     */
+    public function column_block_cols($item): string
+    {
+        $content = $item['block_attrs'] ?? null;
+
+        if (empty($content)) {
+            return '';
+        }
+
+        $cols = $content['columns'] ? count($content['columns']) : '0';
```

Comment:
What do you think about this suggestion? I think you need to cast only within the `if` clause.
```suggestion
               $cols = $content['columns'] ? $content['columns'] : '0';

        if (count($cols) > 4) {
            return "<span style='color: red;'>$cols - Fix needed!</span>";
        }

        return $cols;
```

Reply:
Unless I'm wrong, with your suggestion we will be returning `$cols` as an array (equal to `$content['columns']`) but we need to return the number of elements of the array (`count($content['columns'])`)